tutorials/colors/colormapnorms.py

```diff
@@ -274,17 +276,46 @@
 # Simple geographic plot, set aspect ratio beecause distance between lines of
 # longitude depends on latitude.
 ax.set_aspect(1 / np.cos(np.deg2rad(49)))
+ax.set_title('TwoSlopeNorm(x)')
 fig.colorbar(pcm, shrink=0.6)
 plt.show()
 
 
+###############################################################################
+# FuncNorm: Arbitrary function normalization
+# ------------------------------------------
+#
+# If the above norms do not provide the normalization you want, and you
+# can express your normalization in terms of function, you can use
```

Comment:
I assume you can express any reasonable normalization as a function. So this condition can be dropped?

Reply:
fair enough